saleor/graphql/payment/mutations.py

```diff
@@ -1157,7 +1158,11 @@ class TransactionUpdate(TransactionCreate):
     class Arguments:
         id = graphene.ID(
             description="The ID of the transaction.",
-            required=True,
+            required=False,
+        )
+        token = UUID(
+            description="The token of the transaction to process.",
```

Comment:
I think this description would be better, because the mutation is named `TransactionUpdate`
```suggestion
            description="The token of the transaction to update.",
```